package/MDAnalysis/analysis/density.py

```diff
@@ -394,8 +395,16 @@ def _prepare(self):
                 logger.warning(msg)
             # Generate a copy of smin/smax from coords to later check if the
             # defined box might be too small for the selection
-            smin = np.min(coord, axis=0)
-            smax = np.max(coord, axis=0)
+            try:
+                smin = np.min(coord, axis=0)
+                smax = np.max(coord, axis=0)
+            except ValueError as err:
+                msg = ("No atoms in selection at current frame,"
```

Comment:
As per the other `msg` comment I left, this probably needs to be more informative to users. What does this mean, what are you defaulting to?

This may not be intended behaviour so you need to make sure that users know this is happening and why.

Reply:
"defaulting to user defined grid" - is this better?

Reply:
It depends on the context of the full sentence, but probably not? You'll always end up using the user defined grid, so just saying you are "defaulting" to it doesn't make sense.

See my example above, re: an explicit user message.